chemicaltoolbox/datatypes/molecules.py

```diff
@@ -449,6 +449,33 @@ def set_peek( self, dataset, is_multi_byte=False ):
             dataset.peek = 'file does not exist'
             dataset.blurb = 'file purged from disk'
 
+class PDBQT(GenericMolFile):
+    """
+    Autodock Format
+    The PDBQT format stores the atomic coordinates, partial charges and AutoDock atom types,
+    for both the receptor and the ligand.
+    http://autodock.scripps.edu/faqs-help/faq/what-is-the-format-of-a-pdbqt-file
+    """
+    file_ext = "pdbqt"
+
+    def sniff(self, filename):
+        if count_special_lines('COMPND', filename) > 0:

```

Comment:
My assumption is that this sniffer will also sniff PDB as PDBQT format.
Am I right that the last (non-empty line) needs to be `TORSDOF`? Also ROOT, ENDROOT and BRANCH is unique to PDBQT and do not appear in PDB isn't it?

If we integrate this format in Galaxy we can add a small test file as well and Galaxy will test the sniffer for us. This is definitely the way forward I would say.